packages/app/.env.development

```diff
@@ -14,6 +14,7 @@ MONGO_URI="mongodb://mongo:27017/growi"
 ELASTICSEARCH_URI="http://elasticsearch:9200/growi"
 ELASTICSEARCH_REQUEST_TIMEOUT=15000
 ELASTICSEARCH_REJECT_UNAUTHORIZED=false
+ELASTICSEARCH_REINDEX_ON_BOOT=true
```

Comment:
pls delete this too. same reason(as below comment) in development mode as well.
https://github.com/weseek/growi/pull/5073/files#r788357741